NeuroRuler/__init__.py

```diff
@@ -43,7 +51,12 @@ def gui() -> None:
 
     parser.parse_config_json()
     parser.parse_gui_cli()
-    main.main()
+    guimain.main()
+
+
+def CLI() -> None:
+    """Alias for cli()."""  # TODO: why do we need this?
+    cli()
```

Comment:
Don't have to, but it allows the user to type `cli()` or `CLI()`.

Reply:
[less is more](https://en.wikipedia.org/wiki/KISS_principle) imo

Reply:
I agree, keep only lowercase.